tests/endpoint_decorators_test.py

```diff
@@ -42,3 +42,38 @@ def test_log_headers(*args, **kwargs):
                 "Authorization: REDACTED\n"
                 "authorization: REDACTED\n"
                 "SafeStuff: This better be logged this way")
+
+    def test_logbodies_errors_on_invalid_json(self):
+        """
+        """
+        data = "{\"invalid\": ...}"
+        valueErrorStr = "No JSON object could be decoded"
+        logger_mock = Mock()
+        logger_mock.info = Mock()
+        logger_mock.exception = Mock()
```

Comment:
Oh! The test above the one I just made had `logger_mock.info` set to `Mock()` so I figured you had to... Fixed. 81dc83a